raiden/transfer/channel.py

```diff
@@ -126,23 +130,42 @@ class UnlockGain(NamedTuple):
     from_partner_locks: TokenAmount
 
 
-def get_sender_expiration_threshold(lock: LockType) -> BlockNumber:
-    """ Returns the block number at which the sender can send the remove expired lock.
+def get_safe_initial_expiration(
+    block_number: BlockNumber, reveal_timeout: BlockTimeout
+) -> BlockExpiration:
+    """ Returns the upper bound block expiration number used by the initiator of
+    a transfer or a withdraw. """
+    return BlockExpiration(block_number + reveal_timeout * 2)
 
-    The remove lock expired message will be rejected if the expiration block
+
+def get_sender_expiration_threshold(expiration: BlockExpiration) -> BlockExpiration:
+    """ Returns the block number at which the sender can send:
```

Comment:
nitpick: This should be a one liner.

Suggestion: `Compute the block at which an expiration message can be sent without worrying about blocking the message queue.`